_posts/2021-08-23-rocket-validator.md

```diff
@@ -0,0 +1,119 @@
+
+Site validation reports can also be [scheduled](https://docs.rocketvalidator.com/scheduling/) to run daily, weekly or monthly, thus providing constant monitoring on your sites. For Elixir School we've set a weekly schedule on the site, which is sometimes changed to daily when we're working intensively on fixing the site.
+
+We're also using [deploy hooks](https://docs.rocketvalidator.com/deploy-hooks/) - so a new deploy triggers a site validation report. This was very easy to integrate into Heroku with the [Heroku post-hook add-on](https://devcenter.heroku.com/articles/deploy-hooks#http-post-hook).
+
+## Muting issues we won't fix
+
+Validating a site means checking that the web pages adhere to the current web standards, but there are times when a site can't fully meet the strictest standards.
+
+For example, some of the generated code may be out of your scope. In our case, some HTML generated by Phoenix is not standard HTML (for example, properties like `phx-click` or `phx-track-static` are not valid HTML). We're also using Alpine, which makes use of non-standard properties like `x-data` or `x-cloak`.
+
+Once you're aware of those issues and you've considered if a standard alternative is possible, you can decide to just mute those issues. Rocket Validator lets you mute selected issues and prompts you to document the reason for muting them.
+
+Here's the [list of muted issues on Elixir School](https://rocketvalidator.com/domains/beta.elixirschool.com?tab=mutings&auth=e536facf-2cba-4288-ba45-3e7b95addcf8).
+
+## Beyond automated validation
+
+Automated site validation is a powerful tool to find well-known issues on large sites, and it can quickly point to a lot of typical issues with known solutions. Think of it as a scanner that will find "low-hanging fruit" issues on your site - Rocket Validator can detect [almost 100 typical accessibility issues](https://rocketvalidator.com/accessibility-validation) via axe-core, while the [HTML validator](https://rocketvalidator.com/html-validation) can detect many more possible misuses of web standards.
+
+However, manual checking and common sense will always be required in web development. A site may follow the strictest web standards but still not be accessible. Think, for example, of `alt` descriptions in images - they're not useful at all unless they really provide a [meaningful description](https://duckduckgo.com/?q=meaningful+alt+image+description&t=opera&ia=web).
+
+Apart from automated tests, you also need to do manual checks on your site using a screen reader, turning the screen off, navigating using the keyboard instead of the mouse, etc. We're aware of that and have already a list of [issues discovered via manual testing](https://github.com/elixirschool/school_house/issues/114) that will soon be addressed.
+
+## Conclusion
+
+Web page validation is an important step in web development, that is often skipped because manually checking each web page is a cumbersome task.
```

Comment:
```suggestion
Web page validation is an important step in web development that is often skipped because manually checking each web page is a cumbersome task.
```